Games/types/Mafia/roles/cards/BlockTargets.js

```diff
@@ -0,0 +1,21 @@
+const Card = require("../../Card");
+const { PRIORITY_NIGHT_ROLE_BLOCKER } = require("../../const/Priority");
+
+module.exports = class BlockTargets extends Card {
+  constructor(role) {
+    super(role);
+
+    this.actions = [
+      {
+        priority: PRIORITY_NIGHT_ROLE_BLOCKER,
+        labels: ["block", "hidden"],
+        run: function () {
+          if (this.game.getStateName() != "Night") return;
+
+          let visits = this.getVisits(this.actor);

```

Comment:
@R9H9 nice work on friendly, I used the existing library function for getvisits instead